docs/proposals/service-discovery/README.md

```diff
@@ -49,43 +51,6 @@ Having a `derived-` prefix for `Service` resources seems counterintuitive when t
 
 ## Proposal
 
-Following are flows to support the service import proposal:
-
-1. `Deployment` and `Service` are created on cluster member1 and the `Service` imported to cluster member2 using `ServiceImport` (described below as [user story 1](#story-1))
-2. `Deployment` and `Service` are created on cluster member1 and both propagated to cluster member2. `Service` from cluster member1 is imported to cluster member2 using `ServiceImport` (described below as [user story 2](#story-2))
-
-The proposal for this flow is what can be referred to as local-and-remote service discovery. In the process handling, it can be simply distinguished into the following scenarios:
```

Comment:
Why delete the analysis part of the strategy?

Reply:
I prefer to keep it simple. The analysis part starts from ServiceImport and ServiceExport, which is more complicated and can easily confuse readers.

Reply:
I feel that the two strategies that need to be dealt with still need some explanation. The parts related to ServciceExport and ServiceImport can be removed.

Reply:
Updated, PTAL